firmware/application/apps/ui_adsb_rx.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (C) 2015 Jared Boone, ShareBrained Technology, Inc.
  * Copyright (C) 2017 Furrtek
+ * Copyright (C) 2023 Kyle Reed
```

Comment:
Use "split diff" or ignore the "before". This file was re-ordered to have the same order as the classes in the header to make it easier to find definitions.